components/script/dom/worker.rs

```diff
@@ -162,6 +162,7 @@ impl WorkerMethods for Worker {
         Ok(())
     }
 
+    // https://html.spec.whatwg.org/multipage/#handler-dedicatedworkerglobalscope-onmessage
     event_handler!(message, GetOnmessage, SetOnmessage);
     event_handler!(error, GetOnerror, SetOnerror);
```

Comment:
for cases like this i think. i'll work around it

https://github.com/servo/servo/pull/6699/files#diff-ffcaa7c334fe34498bc7362e376be445R466